src/map.rs

```diff
@@ -172,16 +173,16 @@ where
         h.finish()
     }
 
+    #[inline]
     /// Tests if `key` is a key in this table.
     ///
     /// The key may be any borrowed form of the map's key type, but `Hash` and `Eq` on the borrowed
     /// form must match those for the key type.
-    pub fn contains_key<Q>(&self, key: &Q) -> bool
+    pub fn contains_key<'g, Q>(&self, key: &Q, guard: &'g Guard) -> bool
```

Comment:
This doesn't need `'g`, Just taking `&Guard` should be fine since we don't return a reference.